app/actions/pure/subscribe.js

```diff
@@ -0,0 +1,28 @@
+import { reduxActions } from '../../constants'
+
+export default class SubscribePureActions {
+
```

Comment:
Are we ever going to need to wrap this class in a function? If so we might want to just declare the class and then `export default` at the bottom, but it really doesn't matter haha